### PR TITLE
PR to correct issue #2262

### PR DESCRIPTION
**Pull request by rpedro10**
This PR is supposed to correct the issue:
[https://github.com/zeit/hyper/issues/2262](https://github.com/zeit/hyper/issues/2262)

It was added a flag to check if the Plugins have been loaded and the notifications were changed accordingly so it doesnt notify the user: 'plugin error'  when plugins were added while the app was closed.
Now it says 'Updating Plugins' and not error.

It cant automatically be merged

### CODE REVIEW

File: app/plugins.js [hunk 1]
```diff
@@ -245,7 +254,7 @@ function requirePlugins() {
     } catch (err) {
       //eslint-disable-next-line no-console
       console.error(err);
-      notify('Plugin error!', `Plugin "${basename(path_)}" failed to load (${err.message})`);
+     // notify('Plugin error!', `Plugin "${basename(path_)}" failed to load (${err.message})`);
```
Comment (CodeTheory) [inline app/plugins.js]: Did you mean to leave this commented out?

File: lib/utils/plugins.js [hunk 2]
```diff
@@ -137,10 +137,13 @@ const loadModules = () => {
       } catch (err) {
         //eslint-disable-next-line no-console
         console.error(err.stack);
-        notify(
+        if(plugins.FINISHED_PLUGINS){
+          notify(
           'Plugin load error',
-          `"${pluginName}" failed to load in the renderer process. Check Developer Tools for details.`
+          `"${pluginName}" failed to load in the renderer process, Check Developer Tools for details. `
```
Comment (CodeTheory) [inline lib/utils/plugins.js]: I don't think this grammar is correct.

**Comment by rpedro10** [comment]
I have commited once again regarding your requested changes. The grammar was changed and the commented line was put inside an if clause (without comment)

**Comment by Stanzilla** [comment]
@rpedro10 care to rebase?

**Comment by chabou** [comment]
Sorry but this PR will not be merged. (Already fixed in v2)
Thank you so much for your help 🙏

**Pull request closed without merging**